arch/arm64/core/mmu.c

```diff
@@ -755,6 +780,12 @@ static void invalidate_tlb_all(void)
 	: : : "memory");
 }
 
+static inline void invalidate_tlb_page(uintptr_t virt)
+{
+	/* to be refined */
+	invalidate_tlb_all();
```

Comment:
Looks good to me - this was the only thing that kind of stood out.

Are the necessary bits for more refined tlb invalidation are not yet available?